pyramid_swagger/tween.py

```diff
@@ -202,50 +185,18 @@ def should_exclude_route(excluded_routes, route_info):
     )
 
 
-def _validate_request(route_match, request, schema_data, resolver):
-    """ Validates a request and raises a RequestValidationError on failure.
-
-    :param route_match: a dict with all the path params and their values from
-        the request
-    :param route_match: dict
-    :param request: the request object to validate
-    :type request: Pyramid request object passed into a view
-    :param schema_data: our mapping from request data to schemas (see
-        load_schema)
-    :type schema_data: dict
-    :param resolver: the request object to validate
-    :type resolver: Pyramid request object passed into a view
-    """
+def _validate_request(route_match, request, validator_map):
```

Comment:
https://github.com/striglia/pyramid_swagger/pull/90/files#diff-03bf6185e91ce5bb05dba792eb334610R260

`validation_error()` decorator around `validate_request()` and `validate_response()`